### PR TITLE
fixes "Set average weekly working hour" button in user tab is not responsive Fixes #7309

### LINKED ISSUE #7309: "Set average weekly working hour" button in user tab is not responsive
**Describe the bug**

"Set average weekly working hour" button text is clipping out in user tab ( Mobile View) 

**To Reproduce**

1. Go to https://care.coronasafe.in/users
2. switch to responsive view and see the error

**Expected behavior**

it should be properly responsive in mobile view

**Screenshots**

image

### PR DESCRIPTION
**Pull request by kabirrajsingh**
Fixes  #7309
Removed useless classes in parent component and added necessary classes in child component
Screenshots attached

Screenshot_14
Screenshot_13

### CODE REVIEW

File: src/Components/Users/ManageUsers.tsx [hunk 1]
```diff
@@ -410,18 +410,21 @@ export default function ManageUsers() {
                   {["DistrictAdmin", "StateAdmin"].includes(
                     authUser.user_type
                   ) && (
-                    <div className="flex-col md:flex-row">
+                    <div className="">
                       <ButtonV2
                         id="avg-workinghour"
-                        className="flex w-full items-center md:w-full"
+                        className="flex w-full"
```
Comment (rithviknishad) [inline src/Components/Users/ManageUsers.tsx]:
```suggestion
                        className="w-full"
```

File: src/Components/Users/ManageUsers.tsx [hunk 2]
```diff
@@ -410,18 +410,21 @@ export default function ManageUsers() {
                   {["DistrictAdmin", "StateAdmin"].includes(
                     authUser.user_type
                   ) && (
-                    <div className="flex-col md:flex-row">
+                    <div className="">
```
Comment (rithviknishad) [inline src/Components/Users/ManageUsers.tsx]:
```suggestion
                    <div>
```

**Review by rithviknishad — APPROVED** [review]

**Comment by Ashesh3** [comment]
Deploy is failing

**Review by Ashesh3 — APPROVED** [review]

**Comment by nihal467** [comment]
@kabirrajsingh fix the deploy issue

**Comment by kabirrajsingh** [comment]
@nihal467 it should work now. Apologies for the delay, had exams.

**Comment by nihal467** [comment]
@kabirrajsingh 
image

it is not fixed yet

File: src/Components/Users/ManageUsers.tsx [hunk 3]
```diff
@@ -410,18 +410,20 @@ export default function ManageUsers() {
...
+                        className="w-full"
                         onClick={() => {
                           setExpandWorkingHours(true);
                           setSelectedUser(user.username);
                           setWeeklyHours(user.weekly_working_hours);
                         }}
                       >
                         <CareIcon icon="l-clock" className="text-xl" />
-                        <p>Set Average weekly working hours</p>
+                        <p className="whitespace-nowrap md:whitespace-normal">
```
Comment (rithviknishad) [inline src/Components/Users/ManageUsers.tsx]:
```suggestion
                        <p className="whitespace-normal md:whitespace-nowrap">
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**